### PR TITLE
Add dependabot support to each recipe

### LINKED ISSUE #171: Please update the Fiber version on each go.mod file🐞
I have recently try to test Fiber using several examples, and it was until an hour later I noticed my error was because all the examples on this repo are using a `go.mod` file with old versions of Fiber.

I would be really helpful if all the examples were updated to the most recent version of Fiber.

### PR DESCRIPTION
**Pull request by gaby**
### What this PR fixes:

Currently dependabot can't update dependencies recursively as stated here: https://github.com/dependabot/dependabot-core/issues/2178 https://github.com/dependabot/dependabot-core/issues/2824

This PR adds each directory to dependabot.yml so the go dependencies get updated. 

### Related issue:
Fixes #171 

### How to replicate:

* Git clone this repo: `git clone git@github.com:gofiber/recipes.git`
* Run the following script `python3 create_dependabot.py`

```python3
import os

pwd = os.getcwd()
print(f"Current working directory: {pwd}")

dirs = [entry.name for entry in os.scandir(os.path.join(pwd, "recipes")) if entry.is_dir()]

for entry in dirs:
    print('  - package-ecosystem: "gomod"')
    print(f'    directory: "/{entry}"')
    print('    schedule:')
    print('      interval: "daily"')
```

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,3 +6,259 @@ updates:
...
+  - package-ecosystem: "gomod"
+    directory: "/aws-eb"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/jwt"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/.git"
```
Comment (efectn) [inline .github/dependabot.yml:122]: Remove

File: .github/dependabot.yml [hunk 2]
```diff
@@ -6,3 +6,259 @@ updates:
...
+  - package-ecosystem: "gomod"
+    directory: "/fiber-svelte-netlify"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/optional-parameter"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/.github"
```
Comment (efectn) [inline .github/dependabot.yml:26]: Remove

**Review by efectn — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (gaby) [reply .github/dependabot.yml]: Done

File: .github/dependabot.yml (hunk 2 above)
Comment (gaby) [reply .github/dependabot.yml]: Done

**Comment by gaby** [comment]
I was thinking about ordering the file by folder name. The other idea I had was adding the go.mod files for the folders than dont't have one. Thoughts?

**Review by efectn — APPROVED** [review]

**Comment by gaby** [comment]
@ReneWerner87 When you get a chance, can you take a look at this? :-)

**Comment by ReneWerner87** [comment]
Sure

**Pull request merged**